### PR TITLE
V0.3.0.0

### PR DESCRIPTION
**Pull request by alanz**
Feel free to update the ChangeLog, but there is a *lot* to go in.

### CODE REVIEW

**Comment by bubba** [comment]
[Github search for PRs merged between the last release](https://github.com/haskell/haskell-ide-engine/pulls?utf8=✓&q=is%3Apr+merged%3A2018-07-23..2019+)

File: Changelog.md [hunk 1]
```diff
@@ -1,6 +1,45 @@
...
+   - Suggests modules that can be imported
+   - Suggests GHC extensions
+   - Recognizes when completing a type or expression
+   - Provides snippets for arguments to functions
+ - Add the ability to set an explicit hoogle database
+
+Thanks to the contributors for this release:
+ - bbarker
+ - Avi-D-coder
+ - jhreck
```
Comment (jhrcek) [inline Changelog.md]: jhreck -> jhrcek, but no big deal :D

File: Changelog.md [hunk 2]
```diff
@@ -1,6 +1,45 @@
...
+   - The `--json` flag can be used for JSON transport
+ - HIE now warns you if there is mismatch between the HIE GHC verison and the project GHC version
+ - Add Liquid Haskell support
+ - Add support for hierarchical document symbols
+ - Add many new types of code actions
+   - Typed holes
+   - HaRe refactoring
+   - Misspelled variables
+   - Missing top-level signatures
+   - Prefix unused terms with '_'
```
Comment (jhrcek) [inline Changelog.md]: Using different apostrophe character seems to break markdown formatting. Please use the first one in both places on line 13

**Comment by bubba** [comment]
Sorry, my bad! I should make a script for this one of these days…

**Pull request merged**